functional-tests/common/com.liferay.ide.functional.swtbot/src/com/liferay/ide/functional/swtbot/UI.java

```diff
@@ -421,6 +431,8 @@
 
 	public final String LAYOUT_TEMPLATE = "layout-template";
 
+	public final String LAYOUT_TEMPLATE_PREFERENCE = "Layout Template";
```

Comment:
Check what we already have, and use existing ones, for example, remove "PORTLET_PREFERENCE", use "PORTLET_UPCASE" instead.